pkg/operator/apiserver/audit/audit_policies.go

```diff
@@ -83,11 +83,26 @@ func GetAuditPolicy(audit configv1.Audit) (*auditv1.Policy, error) {
 	p := basePolicy.DeepCopy()
 	p.Name = string(audit.Profile)
 
-	extraRules, ok := profileRules[audit.Profile]
+	for _, cr := range audit.CustomRules {
+		profileRules, ok := profileRules[audit.Profile]
+		if !ok {
+			return nil, fmt.Errorf("unknown audit profile %q", audit.Profile)
```

Comment:
could you change the error msg to indicate that a profile wasn't found in a custom rule?

Reply:
how this error will be displayed to an end-user?

Reply:
This error is never shown to the user. I.e. it should not happen if we get the API validation right.

Reply:
will we have a validation webhook for the audit policies?

Reply:
done via openapi.